### DISCUSSION TITLE
Changing password

### DISCUSSION THREAD

**Discussion by photoneffect** [koush/scrypted#287] [General]
I have access to the Scrypted Management Console, but can't see anywhere on how to change (or remove) the password.  Is the only option the environment variable reset?

**Comment by photoneffect** [comment]
Nevermind.  Realized I can change it when logging in.

**Comment by koush** [comment]
yes during login

**Comment by HOMESEERDUDE** [comment]
I tried reinstalling the container and still can't reset the username and password.

**Reply by bjia56** [reply]
https://github.com/koush/scrypted/wiki/Lost-Password-and-User-Reset

**Comment by HOMESEERDUDE** [comment]
Thank you for getting back to me. I really appreciate it. I'm new at this.
Where exactly does the script go?   In the container?

Thank you again, I was up all night trying to figure it out.

**Reply by bjia56** [reply]
It goes into the scrypted data volume directory, which is usually `~/.scrypted/volume` on the host or `/server/volume` inside the container

**Comment by HOMESEERDUDE** [comment]
So I create a new directory line item there or put the code in an existing
one?

**Reply by bjia56** [reply]
Run the `touch` command listed in the wiki page outside of the container, then restart the container. No config changes needed

**Comment by HOMESEERDUDE** [comment]
You mean ssh into the nas?

**Comment by HOMESEERDUDE** [comment]
Am I in the right place?

**Comment by HOMESEERDUDE** [comment]
Here?

**Comment by HOMESEERDUDE** [comment]
I apologize,  I'm missing something

**Reply by bjia56** [reply]
What machine did you install scrypted on (OS, architecture)? And what method of installation, local or docker compose?

**Comment by HOMESEERDUDE** [comment]
a docker container in a synology nas 723+

**Reply by bjia56** [reply]
Then yes you'll want to run the `touch` command on the nas

**Comment by HOMESEERDUDE** [comment]
Did you see the pictures I sent you?   I'm getting error messages. I'm sure
I'm missing something simple

**Reply by bjia56** [reply]
No, the pictures are not showing on Github. You might need to post them on the Github website instead of via email

**Comment by HOMESEERDUDE** [comment]
Am I entering the command wrong?

**Comment by HOMESEERDUDE** [comment]
I also tried deleting and reinstalling the image and that didn't work
either.

**Reply by bjia56** [reply]
Are you in the Scrypted Discord server? It may be easier to share your screenshots/pictures there, since I'm not seeing anything on Github

**Comment by HOMESEERDUDE** [comment]
What section should it be posted in?

**Reply by bjia56** [reply]
`#docker`
